cves/CVE-2016-8743.yml

```diff
@@ -205,8 +227,10 @@ lessons:
     applies:
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: |
+      Because whitespace may often be introduced into requests without humans noticing,
+      these complex whitespace patterns need not be malicious. They need sanitized nevertheless.
```

Comment:
```suggestion
      these complex whitespace patterns need not be malicious. They need sanitization nevertheless.
```

Small typo here :)

Reply:
Don't think this is a typo. I meant "sanitized"